src/components/MediaUploadZone.tsx

```diff
@@ -0,0 +1,134 @@
+import { UploadSimple, WarningCircle } from "@phosphor-icons/react";
+import { useState, useEffect, useRef } from "react";
+
+interface MediaUploadZoneProps {
+  onFilesDropped: (files: File[]) => void;
+  fontSize?: string;
+  className?: string;
+}
+
+const MediaUploadZone: React.FC<MediaUploadZoneProps> = ({
+  onFilesDropped,
+  fontSize = "1em",
```

Comment:
I do not think we need to pass font size explicitly like this, can you remove it and tell me what it break?